search-app/src/cmr/search/results_handlers/metadata_results_handler.clj

```diff
@@ -109,6 +111,13 @@
                                (when data
                                  (x/element :data {} (json/generate-string data))))))))]))
 
+(defn xml-escape-umm-json-metadata
+  "Returns the metadata. Xml escape the special characters in UMM JSON metadata if applicable."
+  [format metadata]
+  (if (mt/umm-json? format)
+    (string/escape metadata {\< "&lt;", \> "&gt;", \& "&amp;"})
```

Comment:
Hrm ... this seems too hand-tooled to me (and thus too fragile). A more robust solution would be to use whatever community standard there is in Clojure for escaping HTML/XML entities ...

Reply:
More "Hrm": I've been taking a quick look at what Clojure projects do, and it seems that most entity escaping is actually happening inside the various web-oriented projects ... which may indicate there is no standard solution for this.

If that does end up being the case, then let's put this in `cmr.common.xml/escape-entities` ... since this definitely a general-purpose function we'll need again, sooner or later.

Reply:
Good point. Moved.